third-party/CMakeLists.txt

```diff
@@ -141,8 +145,14 @@ set(WIN32YANK_X86_SHA256 62f34e5a46c5d4a7b3f3b512e1ff7b77fedd432f42581cbe825233a
 set(WIN32YANK_X86_64_URL https://github.com/equalsraf/win32yank/releases/download/v0.0.4/win32yank-x64.zip)
 set(WIN32YANK_X86_64_SHA256 33a747a92da60fb65e668edbf7661d3d902411a2d545fe9dc08623cecd142a20)
 
+if(USE_BUNDLED_WINPTY)
+set(WINPTY_URL https://github.com/zhaozg/winpty/archive/master.zip)
+set(WINPTY_SHA256 8da051e6241a57959c9b6ac1812b42627102386dbd8625db1d1c263286694740)
```

Comment:
Can you leave this for a separate PR? Will need to discuss/think about it. We should probably start building winpty ourselves instead of adding more third-party binary sources.